gamedata.fbs

```diff
@@ -1,6 +1,5 @@
 include "comms.fbs";
-include "gamestate.fbs";
-include "matchstart.fbs";
+include "matchconfig.fbs";
 include "rendering.fbs";
```

Comment:
Are all of these needed?

Reply:
looks like you're right, only `matchconfig.fbs` is needed